django/contrib/staticfiles/templatetags/staticfiles.py

```diff
@@ -1,16 +1,29 @@
+import warnings
+
 from django import template
-from django.contrib.staticfiles.storage import staticfiles_storage
 from django.templatetags.static import StaticNode
+from django.templatetags.static import static as _static
+from django.utils.deprecation import RemovedInDjango20Warning
 
 register = template.Library()
 
 
 def static(path):
-    return staticfiles_storage.url(path)
+    # Implementation has been moved to django.templatetags.static.
```

Comment:
It might be worth delaying this deprecation for a few releases, otherwise I fear it could become quite annoying if not impossible for third-party packages to adjust their templates to run deprecation warning free and support older versions of Django.